src/Symfony/Component/String/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Add `TruncateMode` enum to handle more truncate methods
  * Add the `AbstractString::kebab()` method
+ * Add the `AbstractString::pascal()` method
```

Comment:
This will probably not make it into 7.2 as we are already 10 days after the feature freeze.

Reply:
I see..so what can I do?